pkg/apis/kops/cluster.go

```diff
@@ -241,6 +241,11 @@ type ClusterSpec struct {
 
 	// Tags for AWS instance groups
 	CloudLabels map[string]string `json:"cloudLabels,omitempty"`
+
+	// vSphere required specs
+	VSphereServer       string
```

Comment:
This struct does not seem to have anything cloud specific. This is cluster specific struct. Not sure if this is the right place for VC specific things.